front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -129,7 +129,9 @@ const IdeasShowPage = () => {
                   ideaId={idea.data.id}
                   projectId={idea.data.relationships.project.data.id}
                   compact={isSmallerThanTablet}
-                  hasNextPreviousControl={phaseContext !== null}
+                  // TODO: Since next/previous controls were temporarily removed, we set this to false
+                  // for the time being until they are re-introduced.
+                  hasNextPreviousControl={false}
                 />
```

Comment:
I see what happened now - when the next/previous buttons were removed for performance reasons, we forgot to change this behaviour. The code was expecting there to be the next/previous buttons in the UI, but there weren't so it resulted in the incorrect margin.

Added a comment for better clarity as well :)